CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 2.67.0 (Unreleased)
 
+ENHANCEMENTS:
+
+* `azurerm_app_service` - support for the `acr_use_managed_identity_creds` and `acr_user_managed_identity_id` properties [GH-12277]
+
```

Comment:
Hi @mozts2005 - Could you remove this from your PR please? Maintainers update CHANGELOG.md when a PR is merged so that the notes are correctly listed with the appropriate release.

Thanks!

Reply:
all fixed.